sdk/communication/azure-communication-callingserver/azure/communication/callingserver/_server_call.py

```diff
@@ -33,49 +38,171 @@ def __init__(
+
+        play_audio_request = PlayAudioRequestConverter.convert(
+            audio_file_uri, play_audio_options)
 
         return self._server_call_client.play_audio(
             server_call_id=self.server_call_id,
             request=play_audio_request,
             **kwargs
         )
 
+    @distributed_trace()
+    def start_recording(
+        self,
+        server_call_id,  # type: str
+        recording_state_callback_uri,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> StartCallRecordingResult
+
+        if not server_call_id:
+            raise ValueError("server_call_id cannot be None")
+
+        start_call_recording_request = StartCallRecordingRequest(
+            recording_state_callback_uri=recording_state_callback_uri,
+            **kwargs
+        )
+
+        return self._server_call_client.start_recording(
+            server_call_id=self.server_call_id,
+            request=start_call_recording_request,
+            **kwargs
+        )
+
+    @distributed_trace()
+    def pause_recording(
+        self,
+        server_call_id,  # type: str
+        recording_id,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> AsyncHttpResponse
+
+        if not server_call_id:
+            raise ValueError("server_call_id cannot be None")
+
+        if not recording_id:
+            raise ValueError("recording_id cannot be None")
+
+        return self._server_call_client.pause_recording(
+            server_call_id=self.server_call_id,
+            recording_id=recording_id,
+            **kwargs
+        )
+
+    @distributed_trace()
+    def resume_recording(
+        self,
+        server_call_id,  # type: str
+        recording_id,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> AsyncHttpResponse
+
+        if not server_call_id:
+            raise ValueError("server_call_id cannot be None")
+
+        if not recording_id:
+            raise ValueError("recording_id cannot be None")
+
+        return self._server_call_client.resume_recording(
+            server_call_id=self.server_call_id,
+            recording_id=recording_id,
+            **kwargs
+        )
+
+    @distributed_trace()
+    def stop_recording(
+        self,
+        server_call_id,  # type: str
+        recording_id,  # type: str
+        **kwargs  # type: Any
+    ):  # type: (...) -> AsyncHttpResponse
+
+        if not server_call_id:
+            raise ValueError("server_call_id cannot be None")
+
+        if not recording_id:
+            raise ValueError("recording_id cannot be None")
+
+        return self._server_call_client.stop_recording(
+            server_call_id=self.server_call_id,
+            recording_id=recording_id,
+            **kwargs
+        )
+
+    @distributed_trace()
+    def get_recording_properities(
+        self,
+        server_call_id,  # type: str
+        recording_id,  # type: str
+        **kwargs  # type: Any
```

Comment:
We need to add recording apis  _server_call_async.py as well.

Reply:
I added them on my review: https://github.com/Azure/azure-sdk-for-python/pull/20749